site/src/pages/docs/advanced-saml.mdx

```diff
@@ -56,14 +56,14 @@ attach it to your <type://Server>.
 ```java
 // Specify information about your keystore.
 // The keystore contains two key pairs, which are identified as 'signing' and 'encryption'.
-KeyStoreCredentialResolver credentialResolver =
+CredentialResolver credentialResolver =
```

Comment:
I'm guessing this is a mistake since `CredentialResolver` doesn't exist

```suggestion
KeyStoreCredentialResolver credentialResolver =
```

Reply:
Then I think we need to change this one too. :) 
https://github.com/line/armeria/blob/3d800a2be414723d9c49abfd9db95c037e822559/examples/saml-service-provider/src/main/java/example/armeria/server/saml/sp/Main.java#L55

Reply:
Oops nevermind, I was looking at the wrong part of the code. The fix looks good 👍